decide/src/main/java/group17/LicAnalyzer.java

```diff
@@ -271,7 +271,31 @@ public boolean lic10(InputHandler input) {
 
     }
 
+
+    
+    /** 
+     * This function calculates Launch Interceptor Condition (LIC) number 11
+     * 
+     * @param input
+     * @return boolean
+     */
     public boolean lic11(InputHandler input) {
+        int gpts = input.G_PTS;
+        int numpoints = input.NUMPOINTS;
+        double[] X = input.X_COORD;
+
+        if (gpts < 1) {
+            throw new IllegalArgumentException("Exception thrown from: LIC 11. Reason: G_PTS must be greater than 1.");
+        } else if (gpts > numpoints - 2) {
+            throw new IllegalArgumentException("Exception thrown from: LIC 11. Reason: G_PTS must be less than NUMPOINTS - 2.");
+        }
+
+        for (int i = 0; i < numpoints - gpts; i++) {
+            if (X[i + gpts] < X[i]) {
+                return true;
+            }
+        }
```

Comment:
Your implementation does not check for two points separated by G_PTS. For example, if the input data is as follows
- input.NUMPOINTS = 4
- input.G_PTS = 2
- input.X_COORD = [1, 2, 3, 4]
- input Y_COORD = [0, 0, 0, 0]

On the first iteration of the for loop `i = 0`
`X[i] = X[0] = 1` and `X[i + gpts] = X[0 + 2] = 3`
[1, -, 3, 4] - it's only separated by 1 point and not 2

Instead you should compare `X[i]` to `X[i + gpts + 1]` That would make it
`X[i] = X[0] = 1` and `X[i + gpts + 1] = X[0 + 2 + 1] = 4`
[1, -, -, 4] i.e separated by 2

Furthermore, you will need to change your loop end condition to `i < numpoints - gpts - 1`